db/migrate/20160627225519_add_accepted_at_to_auctions.rb

```diff
@@ -0,0 +1,14 @@
+class AddAcceptedAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :accepted_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET accepted_at = delivery_due_at;
```

Comment:
Does this set accepted_at even for auctions that were not accepted? Maybe add a `where accepted = true`